src/main/java/com/spotify/dns/DnsSrvResolvers.java

```diff
@@ -79,7 +86,7 @@ public DnsSrvResolver build() {
         // or if that's empty, localhost.
         resolver = servers == null ?
                    new ExtendedResolver() :
-                   new ExtendedResolver(servers.toArray(new String[servers.size()]));
+                   new ExtendedResolver(servers.toArray(new String[0]));
```

Comment:
@ibauersachs do you remember why we changed this?

Reply:
@fdfzcq this tripped me off at first as well. IntelliJ suggest doing it this way:


```
There are two styles to convert a collection to an array: either using a pre-sized array (like c.toArray(new String[c.size()])) or using an empty array (like c.toArray(new String[0]).

In older Java versions using pre-sized array was recommended, as the reflection call which is necessary to create an array of proper size was quite slow. However since late updates of OpenJDK 6 this call was intrinsified, making the performance of the empty array version the same and sometimes even better, compared to the pre-sized version. Also passing pre-sized array is dangerous for a concurrent or synchronized collection as a data race is possible between the size and toArray call which may result in extra nulls at the end of the array, if the collection was concurrently shrunk during the operation.
This inspection allows to follow the uniform style: either using an empty array (which is recommended in modern Java) or using a pre-sized array (which might be faster in older Java versions or non-HotSpot based JVMs).
```

Reply:
@tommyulfsparre Thank you, by older java we mean pre Java 8 or? :)

Reply:
nvm we think this should be fine!

Reply:
Pre OpenJDK 6 :)